src/Qwiq.Core/IQueryFolder.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Microsoft.Qwiq
+{
+    public interface IQueryFolder : IIdentifiable<Guid>, IEquatable<IQueryFolder>
```

Comment:
Don't we care about other metadata, like path and URI?

Reply:
I don't think so? At least, we don't have a usecase for it yet, and I couldn't easily think of a good one, but that just might be lack of imagination on my part 😃

Reply:
I'll add path as that is common to both soap and rest, but Uri doesn't exist in Soap. We could support it by adding a new IQueryFolder to Rest which includes the added param? Or add it to soap with a NotSupportedException

Reply:
@pelavall I'm fine with omitting it since it has no universal application to both (a URI can be constructed for SOAP, but QWIQ must construct it, rather than having it supplied). You were right: YAGNI will apply here